CONTRIBUTING.md

```diff
@@ -29,7 +29,7 @@ Create a new [free developer instance](https://developer.atlassian.com/platform/
 Create a new [GitHub App](https://github.com/settings/apps), setting the following config:
 
 - **GitHub App name**: Anything you want, but it must be unique across GitHub.
-- **Homepage URL**: `https://DOMAIN`
+- **Homepage URL**: `https://github.com/apps/**GitHub App name**`
 - **Callback URL**: `https://DOMAIN/github/callback`
```

Comment:
Might be nice to update these DOMAIN values too while you are here.(callback, setup....)

You could call out **DOMAIN** above this block (line 30) with a quick sentence what it is 
e.g. The **DOMAIN** value is blah blah blah

Then in the values you could replace them all with `https://[DOMAIN]`